### PR TITLE
refactor filters to use tags.docx

### PR DESCRIPTION
**Pull request by rrusher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Test URLs:
- Before: https://main--vg-volvotrucks-us--hlxsites.hlx.page/news-and-stories/volvo-trucks-magazine/
- After: https://article-tags-list--vg-volvotrucks-us--hlxsites.hlx.page/news-and-stories/volvo-trucks-magazine/

The before and after should be the same. Just switching from using the tags.xlsx to using tags.docx.

### CODE REVIEW

File: blocks/magazine-articles/magazine-articles.js [hunk 1]
```diff
@@ -63,27 +63,30 @@ function buildLatestMagazineArticle(entry) {
...
+  const resp = await fetch('/news-and-stories/tags.plain.html');
+  const markup = await resp.text();
+  const div = document.createElement('div');
+  div.innerHTML = markup;
+  const categoryList = Array.from(div.querySelectorAll('li:nth-child(2) ul:first-child li:first-child li'))
+    .map((li) => li.textContent);
+  const topicList = Array.from(div.querySelectorAll('li:nth-child(2) ul:first-child li:nth-child(2) li'))
+    .map((li) => li.textContent);
+  const truckSeriesList = Array.from(div.querySelectorAll('li:nth-child(2) ul:first-child li:last-child li'))
+    .map((li) => li.textContent);
```
Comment (synox) [inline blocks/magazine-articles/magazine-articles.js:76]: To referer them by position could break easily. I would rather have headings and pull the list after the matching heading. Moreover, the selectors above are difficult to read.

**Review by synox — CHANGES_REQUESTED** [review]
see inline comment about document structure

File: blocks/magazine-articles/magazine-articles.js (hunk 1 above)
Comment (rrusher) [reply blocks/magazine-articles/magazine-articles.js:76]: This is pulling the tags from the [tags.docx](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B42EB92F7-6E8D-4E6D-96AD-F91090A6A072%7D&cid=272ca368-715a-4d1a-8069-93ff3570b7a3) and putting them into dropdowns.

I'm open to ideas on how to address these lists differently.

**Comment by synox** [comment]
@rrusher I don't see it being reverted.

**Review by synox — APPROVED** [review]

**Pull request merged**